### ISSUE TITLE
Login Issues 

### ISSUE THREAD

**Issue by Pallavi961** [nfdz/Cryptool#65]
Hii!!
This is nice work you have done well I am delighted with your product.
Mine is not a problem its just a question I want to ask.....
well, recently my phone was damaged and I can't extract anything from the phone so, can I log on to the new phone without a password ????

**Comment by nfdz** [comment]
Hi! Thanks for your kind words. Of course you can open the app on the new phone without a password.
I'm not sure if you're asking this to restore the date from your previous damaged phone backup, you won't be able to restore the backup if you don't remember the password you had when you made the backup.

**Comment by Pallavi961** [comment]
Yes that is the point i need backup for my account ,can I get backup of my data ???
